### PR TITLE
Completion of Pyqt5 interface of MedAlthea equipped with all the functionalities provided by backend.

### PR DESCRIPTION
**Pull request by rahul007-bit**
## App is ready to get merged with master

Here are some screenshots of the application

image

image

- In the last version application crashes when the user tries to sort the medicine name.
so this bug was solve

image

image

- Add marker for shop and user location too.

- Add pop div when user clicks on the shop marker.
image

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [x] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: apps/resourses.txt [hunk 1]
```diff
@@ -13,4 +13,9 @@ https://loremipsum.io/
 format = '%Y-%m-%d %H:%M:%S'
```
Comment (YadavGulshan) [inline apps/resourses.txt:13]: what's the use of this file?

**Review by YadavGulshan — CHANGES_REQUESTED** [review]
I saw an image in the PR, where it is used?

Can you also provide the screenshots of the frames in description of the PR

File: apps/resourses.txt (hunk 1 above)
Comment (YadavGulshan) [reply apps/resourses.txt]: Hmm, I don't think this file should be included in master branch. I'm removing it.

**Review by YadavGulshan — CHANGES_REQUESTED** [review]
The only thing I need right now, is the proper description of the changes made. 

"Just to be sure that everything is working fine."

**Comment by rahul007-bit** [comment]
> The only thing I need right now, is the proper description of the changes made.
> 
> "Just to be sure that everything is working fine."

What all things should be there in the description??

**Comment by YadavGulshan** [comment]
> 

I need the screenshots of frames developed till now.

**Review by YadavGulshan — APPROVED** [review]

**Pull request merged**